ckanext/cfpb_extrafields/validators.py

```diff
@@ -22,8 +63,65 @@ def positive_number_validator(value):
                 Invalid("Must be a positive number")
         except ValueError:
             Invalid("Must be a positive number")
+    return value
 
+#ckan.logic.validators.isodate(value, context)
+def reasonable_date_validator(value):
```

Comment:
Is there no built-in way to do this?  There are potential holes in the implementation.  This may accept 2015-02-30.